agent/functional_tests/tests/functionaltests_windows_test.go

```diff
@@ -344,3 +353,100 @@ func TestTwoTasksSharedLocalVolume(t *testing.T) {
 	rExitCode := rTask.Containers[0].ExitCode
 	assert.NotEqual(t, 42, rExitCode, fmt.Sprintf("Expected exit code of 42; got %d", rExitCode))
 }
+
+func TestGMSAFile(t *testing.T) {
+	RequireDockerAPIVersion(t, ">=1.30")
```

Comment:
Is this only supported by docker API version >=1.30? Isn't that as long as docker API support securityOpt in hostConfig, this should work.

Reply:
Updated to 1.40 per offline discussion.